### DISCUSSION TITLE
Daily usage report for GitHub Runners.

### DISCUSSION THREAD

**Discussion by an00bhav** [ubicloud/ubicloud#1708] [Ideas]
Are there plans, or is it already possible, to download a day to day report of GitHub action runners usage? 

This would be very helpful in monitoring the usage of the costs per workflow for our org, and to create reports to show reduction of day to day costs for GitHub Actions. 

Also, overall the integration with GitHub Runners was very smooth. Thank you for the great work.

**Comment by byucesoy** [comment]
Hi @anubhavcodes.

You can get some information regarding your usage from the billing page, which  shows consumption per day per instance type, but not per workflow or job.

We might add a more detailed dashboard in the future, but it is not in our plans at the moment. I'm cc'ing @enescakir, who is the owner of our Github Runner integration to ensure that he is aware of this feature request.
